### ISSUE TITLE
Line sometimes stays hidden

### ISSUE THREAD

**Issue by ronnc** [DiZhou92/ngx-line-truncation#23]
https://github.com/DiZhou92/ngx-line-truncation/blob/6423a7b9ac9e70b87160427ce35905cc0a845937/projects/line-truncation-lib/src/lib/line-truncation.directive.ts#L135

Hi there, thanks for the cool library. However, I find that sometimes the element style is still hidden after rendering.

It happens randomly and I don't have a small test case for you but I though it is worth raising it here in case you can see anything obvious.

Could it be possible that clientHeight is <=0 ?

**Comment by DiZhou92** [comment]
it's hard to say the problem without looking at a specific case, can you provide one?

**Comment by DiZhou92** [comment]
close because there was no further conversation

**Comment by connersimmons** [comment]
I think I've been able to replicate this issue:
![Screen Shot 2020-07-30 at 11 44 15 AM](https://user-images.githubusercontent.com/7469758/88944047-41242980-d25a-11ea-8867-729f2a522379.png)

You can see the <p> has text content but also the style is set to hidden still.

**Comment by DiZhou92** [comment]
Hi

Could you provide your issue scenario in a demo project? thanks

Regards,
Di

**Comment by connersimmons** [comment]
@DiZhou92 I've created a StackBlitz with some of the minimum elements from my app to hopefully replicate the issue:
https://stackblitz.com/edit/ionic-dpbvci. I've attached a zip of it as well. Typically the post information is coming from and API, but it was difficult for me to mimic that without giving away some sensitive information. 
[ionic-dpbvci.zip](https://github.com/DiZhou92/ngx-line-truncation/files/5017749/ionic-dpbvci.zip)
